lib/membrane_h264_ffmpeg/parser.ex

```diff
@@ -99,23 +99,28 @@ defmodule Membrane.H264.FFmpeg.Parser do
                 description: """
                 Determines whether to drop the stream until the first set of SPS and PPS is received.
                 """
+              ],
+              max_b_frames: [
+                type: :integer,
+                default: 5,
```

Comment:
According to this FFmpeg's code: https://github.com/FFmpeg/FFmpeg/blob/c6364b711bad1fe2fbd90e5b2798f87080ddf5ea/libavcodec/h264_ps.c#L591
The number of reordered frames is bounded by the `MaxDpbFrames` which is bounded by 16. The actual limit of frames referenced by each frame can be calculated from the profile, more details here: https://en.wikipedia.org/wiki/Advanced_Video_Coding#Decoded_picture_buffering

* As a bare minimum, change the default to 15
* It would be best to get rid of this option and calculate the value based on the profile (this most likely should be added to the pure Elixir version of a parser)